ducktape/__main__.py

```diff
@@ -0,0 +1,131 @@
+from ducktape.tests.test import TestLoader
+from ducktape.tests.test import SerialTestRunner
+from ducktape.tests.test import SimpleStdoutReporter
+from ducktape.tests.test import SimpleFileReporter
+from ducktape.tests.test import TestSessionContext
+from ducktape.cluster.vagrant import VagrantCluster
+
+import os
+import sys
+import random
+import time
+
+
+def mock_test_method():
```

Comment:
Can we just move the mock stuff to an actual mock test, then have a very simple unit test for this? We'll probably want to remove the randomness, but then we could actually test that we're discovering and running the expected set of tests.